css/style.css

```diff
@@ -413,11 +414,38 @@ header .menu-toggle.is-active .bar:nth-child(3) {
     padding: 5px;
   }
 
-  .speakers-list h2 {
+  .speakers-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    margin-top: 30vh;
+    margin-bottom: 30vh;
+    padding: 0 40px;
+  }
+
+  .speaker {
+    display: flex;
+    justify-content: center;
+  }
+
+  .speaker-text {
     font-family: 'Lato', sans-serif;
-    text-align: center;
-    padding-bottom: 40px;
-    font-weight: 700;
+    font-size: 12px;
+    text-align: left;
+    margin-left: 15px;
+    width: 60%;
+  }
+
+  .speaker-text h2 {
+    font-size: 18px;
+    font-weight: bold;
+  }
+
+  .speaker-title {
+    position: absolute;
+    left: 40%;
+    top: 170vh;
+    padding: 20px;
+    border-bottom: #ac3a2d solid 2px;
```

Comment:
- [ ] Kindly fix the position of the speakers heading to stay at the top of the speakers list. This behaivor must have been caused by the `position: absolute`